libandroid-navigation/src/main/java/com/mapbox/navigation/DefaultDirectionsSession.kt

```diff
@@ -56,8 +70,14 @@ class DefaultDirectionsSession(private val router: Router, origin: Point, waypoi
 
     private fun requestRoute() {
```

Comment:
should request `requestRoute` method take listener as a parameter?

Reply:
listener attached in separate method `registerRouteObserver`